sputnik-staking/src/storage_impl.rs

```diff
@@ -76,16 +77,16 @@ impl StorageManagement for Contract {
 
     fn storage_balance_bounds(&self) -> StorageBalanceBounds {
         StorageBalanceBounds {
-            min: U128(User::min_storage() as Balance * env::storage_byte_cost()),
+            min: env::storage_byte_cost().saturating_mul(User::min_storage().into()),
             max: None,
         }
     }
 
     fn storage_balance_of(&self, account_id: AccountId) -> Option<StorageBalance> {
         self.internal_get_user_opt(&account_id)
             .map(|user| StorageBalance {
-                total: user.near_amount,
-                available: U128(user.storage_available()),
+                total: NearToken::from_yoctonear(user.near_amount.0),
```

Comment:
It seems that `user.near_amount` can also be changed to `NearToken` type (borsh and serde serialization for it is the same as U128 that is currently there)